lib/receptor_controller/client/response_worker.rb

```diff
@@ -221,7 +221,7 @@ def queue_opts
       @queue_opts               = {:service  => config.queue_topic,
                                    :auto_ack => config.queue_auto_ack}
       @queue_opts[:max_bytes]   = config.queue_max_bytes if config.queue_max_bytes
-      @queue_opts[:persist_ref] = config.queue_persist_ref if config.queue_persist_ref
+      @queue_opts[:persist_ref] = ENV['HOSTNAME']
```

Comment:
Does `HOSTNAME` exist aleady as an environment variable?

Reply:
Yep it's an implicit one. It gets set in every pod to the container name:

```
[ ⚛ ] crc $ k exec -it tower-collector-dev-5597ccb49-97bd7 -- env | grep HOSTNAME
HOSTNAME=tower-collector-dev-5597ccb49-97bd7
[ ⚛ ] crc $ k exec -it collector-ansible-tower-14bd34fd-1-mqcsc -- env | grep HOSTNAME
HOSTNAME=collector-ansible-tower-14bd34fd-1-mqcsc
```